pages/index.tsx

```diff
@@ -0,0 +1,72 @@
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import Image from 'next/image'
+import styles from '../styles/Home.module.css'
+
+const Home: NextPage = () => {
```

Comment:
Should we move `pages` and `styles` folder into `src` folder to make it's similar to other projects?

Reply:
Moved those folders into `src` https://github.com/Lahphim/react-surveyor-challenge/pull/44/commits/f7bed671df20dc7da86f66821803a5d2dcbd0084